camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/SshCommandSensorYamlTest.java

```diff
@@ -110,7 +110,46 @@ public void testSupressingDuplicates() throws Exception {
                 listener.assertEventCount(1);
             }});
     }
-    
+
+    // "Integration" because takes a couple of seconds
```

Comment:
I wonder why it takes a couple of seconds - it shouldn't be doing any real ssh (because it has a `RecordingsshTool` instead).